selfdrive/ui/translations/main_ja.ts

```diff
@@ -70,7 +70,7 @@
     <message>
         <location line="+0"/>
         <source>leave blank for automatic configuration</source>
-        <translation>空白のままにして、自動設定にします</translation>
+        <translation>自動で設定するには、空白のままにしてください。</translation>
```

Comment:
Do you mind opening a second PR with all these improvements to verify separately? That way it's easier to merge the additions!

Reply:
Ok I will make another PR after this merge is accepted.

Reply:
I have some requests when you open a PR, some translations are bit too long